### PR TITLE
emissions additions

### PR DESCRIPTION
**Pull request by rawadelkontar**

### CODE REVIEW

File: example_project/mappers/Baseline.rb [hunk 1]
```diff
@@ -1073,6 +1073,236 @@ def time_mapping(time)
...
+          'ND': 'MROW',
+          'NV': 'NWPP',
+          'CT': 'NEWE',
+          'DE': 'RFCE',
+          'RI': 'NEWE'}
+
+          #get the state from weather file
+          state = feature.weather_filename.split('_', 1)
+          
+          #apply emissions inputs based on the state
```
Comment (kflemin) [inline example_project/mappers/Baseline.rb:1239]: if you can get some of these from the weather file state, do we still need them in the geojson file?

File: example_project/mappers/base_workflow.osw [hunk 2]
```diff
@@ -92,6 +92,17 @@
...
       }
+    },{
+      "measure_dir_name": "add_ems_emissions_reporting",
+      "arguments": {
+        "__SKIP__": true,
+        "future_subregion": "RMPAc",
+        "hourly_historical_subregion": "Rocky Mountains",
+        "annual_historical_subregion": "RMPA",
+        "future_year": 2020,
+        "hourly_historical_year": 2019,
```
Comment (kflemin) [inline example_project/mappers/base_workflow.osw:103]: what historical and future years are available to choose from?

File: example_project/mappers/Baseline.rb [hunk 3]
```diff
@@ -1073,6 +1073,236 @@ def time_mapping(time)
...
+          #California, Carolinas, Central, Florida, Mid-Atlantic, Midwest, New England, New York, Northwest, Rocky Mountains, Southeast, Southwest, Tennessee, and Texas
+          hourly_historical_mapping_hash = 
+          {'FL': 'Florida',
+          'MS': 'SRMVc',
+          'NE': 'Midwest',
+          'OR': '',
+          'CA': '',
+          'VA': '',
+          'AR': '',
+          'TX': '',
```
Comment (kflemin) [inline example_project/mappers/Baseline.rb:1142]: what do all these blank ones mean? what happens if a user chooses a state with no mapping?

File: example_project/mappers/Baseline.rb (hunk 1 above)
Comment (rawadelkontar) [reply example_project/mappers/Baseline.rb]: addressed

File: example_project/mappers/base_workflow.osw (hunk 2 above)
Comment (rawadelkontar) [reply example_project/mappers/base_workflow.osw:103]: I defined these in geojson schema

File: example_project/mappers/Baseline.rb (hunk 3 above)
Comment (rawadelkontar) [reply example_project/mappers/Baseline.rb]: fixed

**Review by kflemin — APPROVED** [review]

File: example_project/example_project_combined.json [hunk 4]
```diff
@@ -790,6 +790,7 @@
         "name": "Residential 4",
         "type": "Building",
         "building_type": "Single-Family Attached",
+        "number_of_stories": 2,
```
Comment (joseph-robertson) [inline example_project/example_project_combined.json:793]: This is pretty confusing. It's required but we don't validate this feature against the schema?

Comment (kflemin) [reply example_project/example_project_combined.json:793]: I know...but for now we do need it for this process here: https://github.com/urbanopt/urbanopt-geojson-gem/blob/develop/lib/urbanopt/geojson/helper.rb#L284.  I will make a note to review this process to see if we could modify it or extract # stories another way, but for now it needs #stories...

**Comment by kflemin** [comment]
integrating this functionality via the test-against-3.4.0 branch. closing this PR now w/o merging.

**Pull request closed without merging**